### ISSUE TITLE
Bug in geom_half_point() for values smaller than 1

### ISSUE THREAD

**Issue by romanhaa** [erocoar/gghalves#8]
Hi there,

I noticed a strange behavior of `geom_half_point()` when plotting dots with values below 1.

Everything is fine here:

```r
iris %>%
ggplot(aes(Species, Sepal.Length)) +
geom_half_point(side = 'l') +
geom_half_boxplot(side = 'r')
```

![image](https://user-images.githubusercontent.com/18103560/84054737-1c2add00-a9b4-11ea-952d-66017fadf9e3.png)

But when I divide the values first, they are spread all over the place but not where they are supposed to be:

```r
iris %>%
mutate(Sepal.Length = Sepal.Length / 100) %>%
ggplot(aes(Species, Sepal.Length)) +
geom_half_point(side = 'l') +
geom_half_boxplot(side = 'r')
```

![image](https://user-images.githubusercontent.com/18103560/84054805-3795e800-a9b4-11ea-9ad8-5824342de2e2.png)

Strangely, I found a workaround by adding `transformation = PositionJitter`. It's not perfect because the jitter is much less than in the original plot but still better than broken :)

```r
iris %>%
mutate(Sepal.Length = Sepal.Length / 100) %>%
ggplot(aes(Species, Sepal.Length)) +
geom_half_point(side = 'l', transformation = PositionJitter) +
geom_half_boxplot(side = 'r')
```

![image](https://user-images.githubusercontent.com/18103560/84055003-7f1c7400-a9b4-11ea-8fad-88cf9c82bb8e.png)

I hope this helps.

Cheers,
Roman

**Comment by erocoar** [comment]
Thanks for pointing this out! It was caused by falling back to `ggplot2::resolution` when `transformation$width` is `NULL`. Using `PositionJitter`, there is no `width` argument at all, so it used the manual calculation. Fixed with ab17fa44f6b6478ed8bd2bad6c4e2ac5ef4b55c9

This also fixes an issue where `range_scale` had less of an effect than it should have. I'll push the change to CRAN in the next week.

**Comment by romanhaa** [comment]
Fantastic, thanks! Greetings from Amsterdam :)
